snap/snapcraft.yaml

```diff
@@ -208,11 +209,11 @@ parts:
   # GNOME's default icon theme, also used by Fedora
   adwaita-icon-theme:
     after: [utils]
-    plugin: autotools
-    autotools-configure-parameters:
-      - --prefix=/
+    plugin: meson
+    meson-parameters: [--prefix=/]
```

Comment:
What do you think about adding `setup` in all `meson-parameters` entries?
(also, I think that the "array style" is ugly... but that's just my personal opinion)

Reply:
I agree it should be done, but I feel it's something that the `meson` snapcraft plugin should do instead, given that's something supported for very long time, and till we don't break it, I'd keep the warning.